api/src/main/java/org/apache/commons/rdf/api/RDFSyntax.java

```diff
@@ -31,7 +31,7 @@
  * and <a href="https://en.wikipedia.org/wiki/TriX_%28syntax%29">TriX</a>.
  * 
  * @see <a href="https://www.w3.org/TR/rdf11-primer/#section-graph-syntax">RDF 1.1 Primer</a>
- *
+ * @see org.apache.commons.rdf.experimental.RDFParser
  */
 public enum RDFSyntax {
```

Comment:
I wouldn't feel beholden to be all that compatible with the previous release, since this is all < release 1.0.